vdev/src/testing/integration.rs

```diff
@@ -145,13 +167,17 @@ struct Compose {
 
 impl Compose {
     fn new(test_dir: PathBuf, env: Environment, network: Option<String>) -> Result<Option<Self>> {
-        let path: PathBuf = [&test_dir, Path::new("compose.yaml")].iter().collect();
+        let path: PathBuf = [&test_dir, Path::new("compose-temp.yaml")].iter().collect();
```

Comment:
I suggest you use `TempFile` for this, as it will automatically be removed when it goes out of scope.